cloud_AWS/terraform/module/examples/multiple-vpc-regions/bucket/_providers.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
```

Comment:
Why underscore in _providers.tf?

Reply:
I thought that's derived from some terraform convention. I will change it.

Reply:
I don't know of such convention, please rename :)

Reply:
Implemented suggestion

Reply:
Resolved

Reply:
cloud_iam/ still contains _providers.tf, though

Reply:
Resolved